datasets/nz/designated_terrorists/crawler.py

```diff
@@ -0,0 +1,91 @@
+from typing import Generator, Dict, Tuple, Optional
+from lxml.etree import _Element
+from normality import slugify
+from zavod import Context, helpers as h
+import re
+
+# It will match the following substrings: DD (any month) YYYY
+DATE_PATTERN = r"\b(\d{1,2} (?:January|February|March|April|May|June|July|August|September|October|November|December) \d{4})\b"
+
+
+def parse_table(
+    table: _Element,
+) -> Generator[Dict[str, Tuple[str, Optional[str]]], None, None]:
+    headers = None
+    for row in table.findall(".//tr"):
+        if headers is None:
+            headers = []
+            for el in row.findall("./th"):
+                headers.append(slugify(el.text_content()))
+            continue
+
+        cells = []
+        for el in row.findall("./td"):
+            for span in el.findall(".//span"):
+                # add newline to split spans later if we want
+                span.tail = "\n" + span.tail if span.tail else "\n"
+
+            # there can be multiple links in the same cell
+            a_tags = el.findall(".//a")
+            if a_tags is None:
+                cells.append((el.text_content(), None))
+            else:
+                cells.append((el.text_content(), [a.get("href") for a in a_tags]))
+
+        assert len(headers) == len(cells)
+        yield {hdr: c for hdr, c in zip(headers, cells) if hdr}
+
+
+def crawl_item(input_dict: dict, context: Context):
+    # aliases will be either a list of size one or None if there is no aliases
+    name, *aliases = (
+        input_dict.pop("terrorist-entity")[0].strip().split("\n\t\t\t\t", 1)
+    )
+
+    # We remove the prefix and split in the commas
+    aliases = aliases[0].replace("Also known as ", "").split(", ") if aliases else []
```

Comment:
It'd be cleaner to just split on `Also known as` once to get name, aliases, and then split aliases on `, ` as you're doing. I don't think we can rely on `\n\t\t\t\t` being there in future